target/linux/generic/patches-4.4/308-mips32r2_tune.patch

```diff
@@ -1,17 +0,0 @@
-Add -mtune=34kc to MIPS CFLAGS when building for mips32r2
```

Comment:
I will add them, the init is crashing with -mtune=34kc then. What commands should I run on the "good" and the "wrong" binary?